zhmccli/_cmd_lpar.py

```diff
@@ -519,6 +519,28 @@ def lpar_console(cmd_ctx, cpc, lpar, **options):
     cmd_ctx.execute_cmd(lambda: cmd_lpar_console(cmd_ctx, cpc, lpar, options))
 
 
+@lpar_group.command('start', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('LPAR', type=str, metavar='LPAR')
+@click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
+              expose_value=False,
+              help='Skip prompt to confirm starting of the LPAR.',
+              prompt='Are you sure you want to start the LPAR ?')
```

Comment:
I don't think we should have an extra confirmation for starting the LPAR. We only should have such prompts on actions that cause a disruption to something that runs (i.e. on stop).

Reply:
Okay